auditlog_ext_nrq/models/auditlog_log_line.py

```diff
@@ -13,21 +13,22 @@ def create(self, vals):
         if vals.get('new_value_text') or vals.get('old_value_text'):
             old_value_text, new_value_text = vals.get(
                 'old_value_text'), vals.get('new_value_text')
+            tz = self.env['res.users'].sudo().browse(SUPERUSER_ID).tz
```

Comment:
Please consider the case where timezone setting is missing, in which case UTC should be applied.

Reply:
HI @yostashiro Amended Changes